PurchasesTests/SubscriberAttributes/SubscriberAttributeTests.swift

```diff
@@ -57,7 +57,6 @@ class SubscriberAttributeTests: XCTestCase {
 
         expect(receivedDictionary["key"] as? String) == key
         expect(receivedDictionary["value"] as? String) == value
-        expect(receivedDictionary["setTime"] as! Date) == now
```

Comment:
I think this is an artifact from a rebase, I'll fix before the final PR is merged

Reply:
https://github.com/RevenueCat/purchases-ios/pull/189/files#diff-9739dbeb09917a58be870452732ebfcaR60